labman/db/tests/test_plate.py

```diff
@@ -92,9 +93,20 @@ def test_search(self):
             Plate.search(plate_notes='interesting', well_notes='write',
                          query_type='UNION'), [plate22, plate23])
 
+    def strip_out_creation_timestamp(self, plates):
```

Comment:
Is this just a hack to make the changes work with existing tests?

Reply:
Yes, the existing tests are rather ridged.

Reply:
Gotcha. I can make an issue to revisit the tests later.